students/alex_skrn/lesson07/html_render.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+    """Base element class."""
+
+    tag = "html"
+    indent = "    "
+
+    def __init__(self, content=None, **kwargs):
+        """content: str or objects subclassed from Element; kwargs: attrs."""
+        if content is not None:
+            self.content = [content]
+        else:
+            self.content = []
+
+        # this will be used by the render method to render the opening tag
+        self.attrs = self.kwargs_to_str(**kwargs)
+
+    def append(self, new_content):
+        """Add str or other objects to content."""
+        self.content.append(new_content)
+
+    def kwargs_to_str(self, **kwargs):
+        """Convert kwargs to a string to be used in the render method."""
+        a_list = []
+        for k, v in zip(kwargs.keys(), kwargs.values()):
+            a_list.append(k)
+            a_list.append(v)
+        return " ".join(['{}="{}"'] * len(kwargs)).format(*tuple(a_list))
```

Comment:
I think this might be what you are trying to go for:
`a_list = [(k, v) for k, v in kwargs.items()]`